conga-aem-definitions/src/main/roles/aem-dispatcher.yaml

```diff
@@ -288,8 +288,8 @@ config:
       # If set to false only the requesting host is all if it is in the hostWhiteList (or this list is empty), and not in the hostBlackList
       allowAllHosts: true
       #hostWhiteList:
-      #- www.host1.com
-      #- www.host2.com
+      #- "https://www.host1.com"
+      #- "https://www.host2.com"
```

Comment:
to demonstrate that there are regexp, we use a regexp the second entry, eg. `http(s)://(www.)?host2.com`

Reply:
the urls inserted into the configuration file are regex-quoted, so it should not work if a regex is given in the config?

Reply:
yes, you're right.